setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name="nomic",
-    version="3.0.32",
+    version="3.1.0",
```

Comment:
Is the public API actually identical after this, or will things be different?

Reply:
Relatedly, is there a release_notes bump we can do here?

Reply:
Hmm good point, i'm not sure so i'll just do 3.033 for now. I'm guessing a lot of this may change with lance so we can do a major bump then.

Reply:
Ideally, none of this code should really change how the user is using nomic client. Hopefully, it just makes the package more stable